package.json

```diff
@@ -96,6 +94,7 @@
     ]
   },
   "dependencies": {
+    "decode-uri-component": "^0.2.2",
```

Comment:
Why are we adding this as a new dependency? Can we just bump the version with `npm update` if it's a transitive dependency?

Reply:
Unfortunately, that didn't work 😬 The only way I was able to get the updated version was making it a top-level dep.

Reply:
Done!

Reply:
I see. Now that it is updated in the package-lock, can we remove it from here?